README.md

```diff
@@ -271,6 +271,10 @@ bslib is actually published to npm under the name [@rokucommunity/bslib](https:/
 ropm install bslib@npm:@rokucommunity/bslib
 ```
 
+If you're using a version of NodeJS that doesn't support npm aliases, then you can also simply install @rokucommunity/bslib directly without the alias, and BrighterScript will recognize this as well.
```

Comment:
Why even recommending the alias-based approach if the simple install works as well?

Reply:
Oh ok, then maybe you should say that "If you use Node 12+" it is "recommended" to install using the alias.

Reply:
PS: Generally whitespace in the readme is a bit random.
Sometimes there is a blank line around titles, sometimes not.
My OCD, it is hurt.

Reply:
@elsassph I tweaked the readme to be a little more clear. Do you like that any better? 

I really like using `bslib_` as a prefix. `rokucommunity_bslib_` is really verbose, and kind of gross to look at (even though yes, I know the transpiled code isn't exactly pretty anyway...). I could be swayed to remove support for the `bslib` alias entirely if @georgejecook and @chrisdp  and @triwav also agree it's not a big deal to keep the `bslib_` shorter alias. 

I also added some whitespace before the headers. I've been sloppy with that, sorry!

Reply:
I'd prefer `bslib_` as well. If I'm understanding correctly they have to install it themselves vs having ropm do it for them @TwitchBronBron?

Reply:
I say keep the short alias

Reply:
Ok cool. I'll keep the logic as-is, (look for the short alias first, but support the full version if present to support older versions of node). Thanks!